packages/react-components/src/components/RichTextEditor/Plugins/CopyPastePlugin.tsx

```diff
@@ -11,6 +11,8 @@ export default class CopyPastePlugin implements EditorPlugin {
   // don't set value in constructor to be able to update it without plugin recreation
   /* @conditional-compile-remove(rich-text-editor-image-upload) */
   onPaste?: (event: { content: DocumentFragment }) => void;
+  /* @conditional-compile-remove(rich-text-editor-image-upload) */
+  onUploadImage?: (image: string, fileName: string) => void;
```

Comment:
What is the image vs fileName here?

Reply:
> What is the image vs fileName here?

Thank you for pointing out this, the naming is not clear enough.
The image should be changed to imageBlobUrl, and the fileName could be changed to imageFileName, which is required by the SDK for uploading images